src/__tests__/01b-create-users.ts

```diff
@@ -134,9 +137,32 @@ describe("username tests", () => {
     await expect(userWallet2.setUsername({ username })).rejects.toThrow()
   })
 
-  // FIXME: failing for some reason
-  // it('sets account status correctly', async () => {
-  //   await AdminOps.setAccountStatus({uid: userWallet2._id, status: 'locked'})
-  //   await expect(userWallet2.status).toBe('locked')
-  // })
+  it("sets account status correctly", async () => {
+    await AdminOps.setAccountStatus({ uid: userWallet2.user._id, status: "locked" })
+    userWallet2 = await getUserWallet(2)
+    await expect(userWallet2.user.status).toBe("locked")
+  })
+
+  it("set 2fa for user0", async () => {
```

Comment:
maybe `generate2fa` instead of `set` here would be better for consistency

Reply:
`generate2fa` function in our codebase just generates a secret, whereas `save2fa` actually sets it. I'll change it to `save2fa`.